package.json

```diff
@@ -24,8 +27,14 @@
   "engines": {
     "node": ">= 8"
   },
-  "sass": "src/stylesheets/wdfnviz.scss",
-  "main": "src/js/main.js",
+  "sass": "src/wdfnviz.scss",
+  "style": "dist/wdfnviz.css",
+  "main": "dist/wdfnviz.js",
+  "module": "dist/wdfnviz.es6.js",
+  "files": [
+    "dist",
+    "src"
+  ],
   "dependencies": {
     "uswds": "^1.6.6"
```

Comment:
I'd like to see us use 2.0.0 beta. Perhaps this could be discussed at planning on Thursday?

Reply:
Works for me. I'll add onto this PR.. need to fix failing test also.